src/index.html

```diff
@@ -10,7 +10,46 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
```

Comment:
```suggestion
      action="#"
```
try to change and find out what is the difference

Reply:
I have found something like this on StackOverflow

action="#" is the same as link to href="#". It doesn't go anywhere but at the top of that page. The action="/" is the same as link to href="/". The single slash / is root web URL

Reply:
and when you submit the form when action='/', we have error 404, and when # - just reload page
also u can use the link from previous task